config/server-vars.yml

```diff
@@ -129,3 +129,5 @@ EDXAPP_LTI_USER_EMAIL_DOMAIN: "lti.openedx.microsoft.com"
 #Cookies API url
 EDXAPP_API_COOKIE_URL: "%%EDXAPP_COOKIE_API_URL%%"
 
+# View live drop down for CMS
+EDXAPP_ENABLE_MULTI_SITE_VIEWLIVE_DROPDOWN: "%%ENABLE_MULTI_SITE_VIEWLIVE_DROPDOWN%%"
```

Comment:
What's the default? You may want to explicitly set an appropriate value in config/stamp/default/bvt.sh

Reply:
yes, the platform is taking the empty strings as false and this will be empty by default